server/embedded/README.md

```diff
@@ -0,0 +1,36 @@
+# embedded
+The LabKey server application can now be started with embedded tomcat.
+
+#### Setup
+1. Uncomment useEmbeddedTomcat, useLocalBuild and useSsl (if you have the local ssl setup, more instructions below) in root level gradle.propeties file
+2. gradle refresh
+3. pickpg/pickmssql
+4. gradle cleanBuild, cleanDeploy and deployApp
+5. Select LabKeyEmbbedded_Dev from IntelliJ run configurations
+
+#### Embedded tomcat gradle properties explained:
++ useEmbeddedTomcat - if present, this will cause the :server:embedded project to be included in your local set of Gradle projects to be built.  This also will affect the behavior of the pickPG and deployApp tasks and is required to be present in order to build a distribution with an embedded Tomcat server. This property is present in the root-level gradle.properties file, but commented out by default
++ useLocalBuild - If present, will allow the embedded server to use the locally built modules from build/deploy/modules.  This property is present in the root-level gradle.properties file, but commented out by default
++ useSsl - If present, the default port for the server will be 8443 instead of 8080 and the server.ssl properties in the application.properties file will be uncommented.  Note, however, that these ssl properties will still need to be updated by hand to reflect your local certificate location, password, etc.
```

Comment:
```suggestion
+ `useSsl` - If present, the default port for the server will be 8443 instead of 8080 and the `server.ssl` properties in the `application.properties` file will be uncommented.  Note, however, that these ssl properties will still need to be updated by hand to reflect your local certificate location, password, etc.
```